lib/proto.js

```diff
@@ -130,7 +130,8 @@ function write (_obj, msg, num) {
     // it is not needed anymore
     stream.lastMsg = msg
     stream.lastObj = obj
-    stream.lastTime = t.slice(8)
+    const timeMatch = t.match(/\d+/)
```

Comment:
We shouldn’t have a regular expression in this hot path. It would be better to use `.slice` with a correct count.

Reply:
Added `timeSliceIndexSym` with correct count.